form.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html" charset="utf-8">
+    <link rel="StyleSheet" href="style.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap&subset=cyrillic" rel="stylesheet">
+    <title>Конкурс PINK</title>
+  </head>
+  <body>
+    <header class="page-header">
+      <nav class="page-header__nav nav">
+        <div class="nav__menu--prewiew">
```

Comment:
```suggestion
        <div class="nav__menu-prewiew">
```
Тут тоже не нужен модификатор. 

Модификаторы используются так: 
`article__title article__title--bold`